elide-contrib/elide-test-helpers/README.md

```diff
@@ -0,0 +1,61 @@
+# Elide Test Helpers
+
+A set of helpers for testing Elide web services
+
+## Installation
+```xml
+<dependency>
+  <groupId>com.yahoo.elide</groupId>
+  <artifactId>elide-test-helpers</artifactId>
+  <version>${elide.verison}</version>
+  <scope>test</scope>
+</dependency>
+```
+
+## Usage
+
+The `JsonApiDSL` can be used to build JSON:API documents in a typesafe way.
+
+```java
+package com.yahoo.elideinstnace.integration
+
+import org.apache.http.HttpStatus;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.*;
+
+public class ReviewTest {
+  @Test
+  public void createReview() {
+
+    given().
+    contentType("application/vnd.api+json").
+    body(
+      data(
+        resource(
+          type("review"),
```

Comment:
not-a-blocker/curiosity: In the long-term can we eventually evolve this so it can take our models? 

This kind of client seems generally useful, however. Any thoughts on contributing this back as a Java client implementation?

Reply:
That is a possibility. Although, I wonder if creating objects would be as readable as the declarative nature of a DSL.